prusti-interface/src/prusti_error.rs

```diff
@@ -13,11 +13,11 @@ use ::log::warn;
 ///
 /// A Prusti message can originate from:
 /// * invalid usage detected during specification collection
-/// * an encoding error (see the `EncodingError` type)
+/// * an encoding error (see the `SpannedEncodingError` type)
 /// * a Viper verification error
 ///
 /// A `PrustiError` can be displayed as a *warning* to the user. (We should rename `PrustiError`,
-/// `EncodingError` and similar types to something less confusing.)
+/// `SpannedEncodingError` and similar types to something less confusing.)
```

Comment:
I think the usual term is "diagnostics".

Reply:
So, `PrustiDiagnostic`, `EncodingDiagnostic` and `SpannedEncodingDiagnostic`? It's 5 characters longer then `...Error`, but I like it because the type alias would hide it. @vakaras @cmatheja any opinion before I do the renaming?

Reply:
Diagnostic in the Rust compiler is the object used to construct the final message to the user. Here, however, it is an error of the computation. Therefore, I would prefer to keep `Error`.

Reply:
I'll keep as it is for the moment.